addons/website_sale_collect/views/delivery_form_templates.xml

```diff
@@ -5,26 +5,27 @@
         <div name="unavailable_products_warning" class="alert alert-warning mt-2">
              Some of the products are not available at <strong><t t-out="wh_name"/></strong>.
              <div
-                 t-foreach="unavailable_products"
-                 t-as="product"
+                 t-foreach="unavailable_order_lines"
+                 t-as="order_line"
                  t-attf-class="d-flex m-2 position-relative"
              >
                  <div class="d-flex align-items-center gap-2">
-                    <a t-att-href="product.website_url">
+                     <a t-att-href="order_line.product_id.website_url">
```

Comment:
:eyes: 
As per my IDE, the `<a>` and the `<t>` tags below are well positioned, maybe you want to fix the `<div>` just above?

```suggestion
                    <a t-att-href="order_line.product_id.website_url">
```
![image](https://github.com/user-attachments/assets/edd8823c-0bd4-440e-8f26-ec98ec006699)